gaurav/vite.config.ts

```diff
@@ -14,6 +14,7 @@ export default defineConfig({
   resolve: {
     alias: {
       "@": path.resolve(__dirname, "./src"),
+      '@shadcn': '/node_modules/@shadcn',
```

Comment:
Why this? its not even being used anywhere

Reply:
Removed